hybrid/compartment_examples/inter_comp_call_secure/main.c

```diff
@@ -0,0 +1,194 @@
+/* In this example, we tighten security even further. For each compartment we
+ * create, we store some capabilities of interest. Essentially, we setup
+ * certain entry points, and make them available to each compartment.
+ * Currently, these involve the PCC/DDC of the `switch_compartment` function
+ * (PCC to execute, DDC to access memory where we store information pertaining
+ * to compartments; essentially, `switch_compartment` performs a limited
+ * privelege escalation, but only over prepared data within `main`).
```

Comment:
sp "privilege"

Reply:
Fixed in https://github.com/capablevms/cheri-examples/pull/55/commits/3955d6af194c6f59f5bc9f71b06702e66ecaa9dc.